accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-blink-rules.ts

```diff
@@ -28,6 +28,8 @@ let a11yRulesBlink: Rule[] = [
         context: "dom:blink",
         run: (context: RuleContext, options?: {}): RuleResult | RuleResult[] => {
             const ruleContext = context["dom"].node as Element;
+            //skip the rule
+            if (RPTUtil.isNodeHidden(ruleContext)) return null;
```

Comment:
This shouldn't be skipped, this is a visual issue

Reply:
removed